src/components/list/List.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
Deze heb je tegenwoordig niet meer nodig, tenzij je state of life cycles gebruikt!